### PR TITLE
*: improve the column info for client

### LINKED ISSUE #4660: OrgTable & Schema is missing from `ResultField`, causes Navicat disabling edit tables
MySQL:

```
mysql> select * from foo;
Field   1:  `ID`
Catalog:    `def`
Database:   `test`
Table:      `foo`
Org_table:  `foo`
Type:       VAR_STRING
Collation:  utf8_general_ci (33)
Length:     60
Max_length: 17
Decimals:   0
Flags:      NOT_NULL PRI_KEY NO_DEFAULT_VALUE PART_KEY 


+-------------------+
| ID                |
+-------------------+
| 12321312312312312 |
| 22222222222221    |
+-------------------+
2 rows in set (0.01 sec)
```

TiDB:

```
mysql> select * from foo;
Field   1:  `ID`
Catalog:    `def`
Database:   ``
Table:      `foo`
Org_table:  ``
Type:       VAR_STRING
Collation:  utf8_general_ci (33)
Length:     80
Max_length: 18
Decimals:   0
Flags:      NOT_NULL PRI_KEY UNIQUE_KEY NO_DEFAULT_VALUE 


+--------------------+
| ID                 |
+--------------------+
| 201709221200018888 |
+--------------------+
1 row in set (0.00 sec)
```

The missing of `Database` and `Org_table` in column meta data causes Navicat < 12 disabling table editing feature.

### PR DESCRIPTION
**Pull request by XuHuaiyu**
fix #4660 
PTAL @coocood @hanfei1991 @breeswish

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
fix #4773

**Comment by XuHuaiyu** [comment]
PTAL @shenli @coocood

File: executor/executor_test.go [hunk 1]
```diff
@@ -1644,6 +1644,22 @@ func (s *testSuite) TestColumnName(c *C) {
 	c.Check(len(fields), Equals, 2)
 	c.Check(fields[0].Column.Name.L, Equals, "d")
 	c.Check(fields[1].Column.Name.L, Equals, "c")
+	rs, err = tk.Exec("select c as a from t as t2")
```
Comment (shenli) [inline executor/executor_test.go]: Add comment for your test case to explain the purpose of the test.

File: expression/schema.go [hunk 2]
```diff
@@ -115,7 +115,9 @@ func (s *Schema) FindColumnAndIndex(astCol *ast.ColumnName) (*Column, int, error
 			if idx == -1 {
 				idx = i
 			} else {
-				return nil, -1, errors.Errorf("Column %s is ambiguous", col.String())
+				if !col.IsAggOrSubq {
```
Comment (shenli) [inline expression/schema.go]: Add comment here.

**Comment by XuHuaiyu** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**